engineering-feedback/feedback-examples.md

```diff
@@ -0,0 +1,43 @@
+# Engineering Feedback Examples
```

Comment:
I like all these real world examples and the various scenarios that resulted in resolution. Helps to cement the feedback concepts with the reader.

Most of these align well with the **Examples of Good Engineering Feedback** section in the readme.md doc. Do you think we need to format them to align with the guidance suggested in the **feedback-guidance.md** doc? Or suggest improvements to the feedback based on the guidance?

I feel like we need to thread a story through each of these docs that ensures the reader is guided towards a consistent outcome.